### ISSUE TITLE
`clearDatabaseRecord` isn't making the record inactive

### ISSUE THREAD

**Issue by PaulWieland** [apearson/insteon-plm#32]
In testing `clearDatabaseRecord`, it seems that the device in the link record is changed to zeros as expected, but the record's active flag is still true (expected false).

Here is what my link record looks like after clearing it:

```
{
            "address": [
                15,
                175
            ],
            "type": 186,
            "Type": {
                "active": true,
                "control": 0,
                "smartHop": 3,
                "highWater": false
            },
            "group": 0,
            "device": [
                0,
                0,
                0
            ],
            "onLevel": 0,
            "rampRate": 0
}
```

I don't yet know if this is an issue with the library, or just an unexpected Insteon behavior.

**Comment by apearson** [comment]
I ran into this.  I think the active flag might be overruled by the highwater flag.  No matter how I set the flag it wouldn't change.  I'll go back and double check the logic to make sure the bit is being set correctly

**Comment by PaulWieland** [comment]
What I've done is put a filter on `link.address !== '[00,00,00]'` to remove cleared links from the Node-RED GUI.

**Comment by apearson** [comment]
This is now fixed, the `||` was causing the false of the active flag to become true

**Comment by apearson** [comment]
Fixed in f65440a
